apps/judicial-system/backend/src/app/formatters/subpoenaPdf.ts

```diff
@@ -71,6 +72,11 @@ export const createSubpoena = (
   doc.on('data', (chunk) => sinc.push(chunk))
 
   setTitle(doc, formatMessage(strings.title))
+
+  if (dateLog) {
```

Comment:
This seems a bit weird but there is a reason for the double if statement in this file. The thing is if we create the stamp first it's difficult to reset the the text position when we start to create the subpoena text. It's easier to first create the space for the stamp, then create the subpoena and finally create the stamp at the top.